docs/ops/comparison/NotEqual_1.md

```diff
@@ -4,7 +4,18 @@
 
 **Category**: Comparison binary operation
 
-**Short description**: *NotEqual* performs element-wise comparison operation with two given tensors applying multi-directional broadcast rules.
+**Short description**: *NotEqual* performs element-wise comparison binary operation with two given tensors applying
+multi-directional broadcast rules.
+
+**Detailed description**
+Before performing comparison operation, input tensors *a* and *b* are broadcasted if their shapes are different and
+`auto_broadcast` attributes is not `none`. Broadcasting is performed according to `auto_broadcast` value.
```

Comment:
please re-write it to:
`Before performing comparison operation, input tensors *a* and *b* are broadcasted if their shapes are different. Broadcasting is performed according to `auto_broadcast` value.`